### DISCUSSION TITLE
vs code

### DISCUSSION THREAD

**Discussion by sudobrat** [prasanthrangan/hyprdots#148] [Q&A] [answered]
i dont know if it is bug related to this but whenever I open vscode it doesnot open last opened folder whereas  it is enabled in the settings

**Comment by T-Crypt** [comment, marked as answer]
Go to: Open File --> Preferences --> Settings and add "window.restoreWindows": "all" to your settings.json file. Remember to add a comma to the preceding statement.

This will reopen all the windows you had open on the close of your last session.

**Comment by sudobrat** [comment]
thanks

**Reply by T-Crypt** [reply]
You can also try adding below if that doesn't work.

"window.reopenFolders": "all"
